platform/platform-app/pom.xml

```diff
@@ -447,7 +448,7 @@
         <cxf.joda.time.version>2.9.4</cxf.joda.time.version>
         <cxf.jettison.version>1.3.8</cxf.jettison.version>
         <cxf.jibx.version>1.2.6</cxf.jibx.version>
-        <cxf.neethi.version>3.0.3</cxf.neethi.version>
+        <cxf.neethi.version>3.1.0</cxf.neethi.version>
```

Comment:
The version numbers in the comments on line 370 and 416 of this file could also be updated

Reply:
Fixed. Thanks for the good eye!